.github/workflows/test-deploy-e2e.yaml

```diff
@@ -1,15 +1,73 @@
-name: CI/CD
+name: Continuous Integration - SDK E2E
```

Comment:
I know I'm being nitpicky 😉 but this is not a `Continuous Integration` action, what about `Deployed environments - SDK E2E tests`?